bullet_train-fields/app/helpers/fields/html_editor_helper.rb

```diff
@@ -5,7 +5,10 @@ def html_sanitize(string)
     return string unless string
     # TODO this is a hack to get around the fact that rails doesn't allow us to add any acceptable protocols.
     string = string.gsub("bullettrain://", TEMPORARY_REPLACEMENT)
-    string = sanitize(string, tags: %w[div br strong em b i del a h1 blockquote pre ul ol li action-text-attachment figure figcaption img], attributes: %w[href sgid content-type url filename filesize width height presentation src class])
+    string = sanitize(string,
+                      tags: %w[div br strong em b i del a h1 blockquote pre ul ol li action-text-attachment figure figcaption img video source],
+                      attributes: %w[href sgid content-type url filename filesize width height presentation src class controls preload type]
+                     )
```

Comment:
Here I added line breaks to make things more readable, and added `video source` to `tags` and `controls preload type` to `attributes`.